### PR TITLE
Feature/revert 42 sls naming

### LINKED ISSUE #43: New upgrade of resources names breaking existing deployments
New minor upgrade `1.8.0` of normalizing resources names breaks current deployments.
Cloudformation failed with CREATE_FAILED.

For example:
Logical ID `BcexplorerdevelopResourcesmetadataAutoScalingPolicyDevUswest2` turned to `ResourcesmetadataAutoScalingTarget` after plugin upgrade. Since `BcexplorerdevelopResourcesmetadataAutoScalingPolicyDevUswest2` already exists and point to same physical id, the new creation of `ResourcesmetadataAutoScalingTarget` fails with 
function:bc-explorer-resources-metadata-develop:provisioned|lambda:function:ProvisionedConcurrency|lambda already exists in stack arn:aws:cloudformation:us-west-2:*****:stack/bc-explorer-develop/bc****

Currently I downgraded version.

### PR DESCRIPTION
**Pull request by cookieMr**
Reverts #42 to fix #43.
Creating version 1.9.0 to address this issue with NMP dash versioning (^1.8.0 should pull 1.9.0).

### CODE REVIEW

**Comment by cookieMr** [comment]
@claydanford @yuvalyacoby
please take a look into this revert

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "serverless-provisioned-concurrency-autoscaling",
-  "version": "1.8.0",
+  "version": "1.7.0",
```
Comment (claydanford) [inline package-lock.json:3]: Should this be 1.9.0?

File: src/utility.ts [hunk 2]
```diff
@@ -4,10 +4,12 @@ export const clean = (input: string): string => {
...
 
+const md5 = (input: string): string => {
+  return createHash('md5').update(input).digest('hex')
+}
+
 export const truncate = (input: string): string => {
-  return input.length <= 64
-    ? input
-    : input.substr(0, 32) + createHash('md5').update(input).digest('hex')
+  return input.length <= 64 ? input : input.substr(0, 32) + md5(input)
```
Comment (claydanford) [inline src/utility.ts:12]: Lets remove the md5 function, and leave this utility as is.

File: package-lock.json (hunk 1 above)
Comment (cookieMr) [reply package-lock.json:3]: this is package-lock.json
but let me run `npm i` + tests locally to update this file

**Review by claydanford — APPROVED** [review]
@cookieMr Thanks for taking care of this!

**Comment by nmrad-91** [comment]
@claydanford do you approve?

File: src/utility.ts (hunk 2 above)
Comment (cookieMr) [reply src/utility.ts:12]: https://github.com/neiman-marcus/serverless-provisioned-concurrency-autoscaling/issues/47

**Pull request merged**